lib/quoters/WebhookQuoter.ts

```diff
@@ -31,6 +39,32 @@ export class WebhookQuoter implements Quoter {
     return QuoterType.RFQ;
   }
 
+  private async getEligibleEndpoints(): Promise<WebhookConfiguration[]> {
+    const endpoints = await this.webhookProvider.getEndpoints();
+    try {
+      const config = await this.circuitBreakerProvider.getConfigurations();
+      const fillerToConfigMap = new Map(config.map((c) => [c.name, c]));
+      if (config) {
+        const enabledEndpoints: WebhookConfiguration[] = [];
+        endpoints.forEach((e) => {
+          if (
+            this.ALLOW_LIST.has(e.name) ||
```

Comment:
wouldn't we still need a way to change the dynamo table even with this allowlist setting? In order to turn the endpoints back on, or else everything will have to be in ALLOW_LIST eventually?

Reply:
hmmmm you are right, we need a way to retire entries in the config, maybe a timestamp

the allow list is still needed tho otherwise they cron would just turn someone off again right after we re-enable